package.json

```diff
@@ -97,6 +97,26 @@
           }
         ],
         "priority": "option"
+      },
+      {
+        "viewType": "one.viewer.metadata",
+        "displayName": "Metadata",
+        "selector": [
+          {
+            "filenamePattern": "*"
+          }
+        ],
+        "priority": "option"
+      },
+      {
+        "viewType": "one.viewer.relation",
+        "displayName": "Relation",
```

Comment:
Could you remove these `Relation Viewer`-related lines? (1)

Reply:
All right, I'll apply that part.